### DISCUSSION TITLE
Can't add a seealso admonition inside a table cell

### DISCUSSION THREAD

**Discussion by trickyck** [pradyunsg/furo#445] [Potential Issue] [answered]
I want to be able to add text wrapped inside a seealso admonition inside a table cell.

This is the raw markdown code:
<img width="535" alt="Screen Shot 2022-05-26 at 6 36 06 pm" src="https://user-images.githubusercontent.com/31978630/170456425-367ae04b-1ba7-45fd-bff0-c63f7d428272.png">

This is how it renders:
<img width="536" alt="Screen Shot 2022-05-26 at 6 36 20 pm" src="https://user-images.githubusercontent.com/31978630/170456460-41e4dc9b-c237-46f5-b73e-73a08d649e6d.png">

When I take out the <br> tag and press <kbd>Enter</kbd>, it creates a new row (I don't want this).

**Comment by pradyunsg** [comment, marked as answer]
That looks like a limitation of the syntax rather than Furo.

Consider using list-table instead, as in https://github.com/pradyunsg/furo/blob/0f3506c7db3641e8a764932addfbbe5be43d9e10/docs/kitchen-sink/tables.rst?plain=1#L109-L119
